pkg/gds/config/config.go

```diff
@@ -88,9 +88,10 @@ type EmailConfig struct {
 }
 
 type CertManConfig struct {
-	Interval time.Duration `split_words:"true" default:"10m"`
-	Storage  string        `split_words:"true" required:"false"`
-	Sectigo  sectigo.Config
+	RequestInterval    time.Duration `split_words:"true" default:"10m"`
+	ReissuenceInterval time.Duration `split_words:"true" default:"24h"`
```

Comment:
```suggestion
	ReissuanceInterval time.Duration `split_words:"true" default:"24h"`
```

Note that this will have to be updated in the other files as well.

Reply:
confirmed